### PR TITLE
Add more fetch keys

### PR DESCRIPTION
**Pull request by autonome**

### CODE REVIEW

File: features/fetch.yml [hunk 1]
```diff
@@ -10,34 +10,54 @@ compat_features:
   - api.Headers.append
   - api.Headers.delete
   - api.Headers.get
+  - api.Headers.getSetCookie
```
Comment (Elchi3) [inline features/fetch.yml:13]: What about these?
- api.Headers.entries
- api.Headers.forEach
- api.Headers.iterate_combined_duplicates
- api.Headers.iterate_lexicographically
- api.Headers.keys
- api.Headers.values

File: features/fetch.yml [hunk 2]
```diff
@@ -10,34 +10,54 @@ compat_features:
...
   - api.Headers.has
   - api.Headers.set
   - api.Request
   - api.Request.Request
+  - api.Request.Request.init_referrer_parameter
```
Comment (Elchi3) [inline features/fetch.yml:18]: What about these other unmapped keys?

- api.Request.Request.cross_origin_stripped
- api.Request.Request.init_attributionReporting_parameter
- api.Request.Request.request_body_readablestream
- api.Request.Request.response_body_readablestream
- api.Request.cache.only-if-cached
- api.Request.credentials.default_same-origin
- api.Request.mode.navigate_mode
- api.Request.targetAddressSpace

File: features/fetch.yml [hunk 3]
```diff
@@ -10,34 +10,54 @@ compat_features:
...
+  - api.Request.referrer
+  - api.Request.referrerPolicy
   - api.Request.text
   - api.Request.url
   - api.Response
   - api.Response.Response
   - api.Response.arrayBuffer
   - api.Response.blob
   - api.Response.bodyUsed
+  - api.Response.body
```
Comment (Elchi3) [inline features/fetch.yml:46]: What about these?

- api.Response.Response.accept_readablestream
- api.Response.Response.body_parameter_optional
- api.Response.body.readable_byte_stream

File: features/fetch.yml [hunk 4]
```diff
@@ -10,34 +10,54 @@ compat_features:
...
   - api.Response.ok
+  - api.Response.redirect_static
+  - api.Response.redirected
   - api.Response.status
   - api.Response.statusText
   - api.Response.text
   - api.Response.type
   - api.Response.url
   - api.fetch
+  - api.fetch.authorization_removed_cross_origin
```
Comment (Elchi3) [inline features/fetch.yml:63]: Should these be considered?

- api.fetch.blob_data_support
- api.fetch.init_attributionReporting_parameter
- api.fetch.init_keepalive_parameter
- api.fetch.init_referrerPolicy_parameter

File: features/xhr.yml [hunk 5]
```diff
@@ -17,6 +17,7 @@ compat_features:
   - api.XMLHttpRequest
   - api.XMLHttpRequest.abort
   - api.XMLHttpRequest.abort_event
+  - api.XMLHttpRequest.authorization_removed_cross_origin
```
Comment (Elchi3) [inline features/xhr.yml:20]: Some more keys that might be relevant?

- api.XMLHttpRequest.getAllResponseHeaders.header_names_lowercase
- api.XMLHttpRequest.responseType
- api.XMLHttpRequest.responseType.arraybuffer_value
- api.XMLHttpRequest.responseType.blob_value
- api.XMLHttpRequest.responseType.document_value
- api.XMLHttpRequest.responseType.json_value
- api.XMLHttpRequest.responseURL
- api.XMLHttpRequest.send.body_URLSearchParams_type
- api.XMLHttpRequest.setAttributionReporting (this one probably belongs to Attribution Reporting)
- api.XMLHttpRequest.setPrivateToken (this one belongs elsewhere, likely?)

**Review by Elchi3 — COMMENTED** [review]
My review approach here is a comparison against the burndown of non-deprecated and standardized BCD keys for the interfaces mentioned and so I found more keys to consider here. But it could also be that some keys want to live in a separate feature or already do, so take the suggestions as inspiration rather than blockers.

**Comment by autonome** [comment]
> My review approach here is a comparison against the burndown of non-deprecated and standardized BCD keys for the interfaces mentioned and so I found more keys to consider here. But it could also be that some keys want to live in a separate feature or already do, so take the suggestions as inspiration rather than blockers.

this is great, thanks! i found the additional keys in this PR... somewhere (?), so added them. definitely will add these others and see what it looks like.

**Comment by autonome** [comment]
Aha i see your other patch now! I'll limit this to fetch, so we don't conflict, and you can do all the xhr additions in #1963.

**Comment by autonome** [comment]
added all, looks good, thanks @Elchi3!

**Comment by autonome** [comment]
Did a pass to see if any of these keys should be separate features.

Caniuse doesn't break any sub features out, just has `Fetch` and shows the baseline high widget.

There's a multi-year trickle of additions for blob support, readable stream integration and misc bits and bobs, some of which are baseline false.

There's nothing that seems to stand out from the implementation timeline, nor from the docs or dev comments/writing based on the searches I did.

There aren't any keys that match the other sub-features (metadata and priority).

So leaving it all in here for now.

**Review by Elchi3 — APPROVED** [review]
Nice, this looks good to me. Great to see all ducks in a row for fetch. I think we can always move stuff out into a separate feature later if there is appetite.

**Pull request merged**